packages/runtime/test/src/interpreter/opcode-list.ts

```diff
@@ -7184,4 +7185,83 @@ describe("Teal Opcodes", function () {
 		});
 	});
 
+	describe("Ed25519verify_bare", function () {
+		const stack = new Stack<StackElem>();
+
+		it("should push 1 to stack if signature is valid", function () {
+			const account = generateAccount();
+			const hexStr = "62fdfc072182654f163f5f0f9a621d729566c74d0aa413bf009c9800418c19cd";
+			const data = Buffer.from(
+				hexStr,
+				"hex"
+			);
+			const signature = signBytes(data, account.sk);
+
+			stack.push(data); 
+			stack.push(signature); 
+			stack.push(decodeAddress(account.addr).publicKey);
+
+			const op = new Ed25519verify_bare([], 1);
+			op.execute(stack);
+			const top = stack.pop();
+			assert.equal(top, 1n);
+		});
+
+		it("should push 0 to stack if signature is invalid", function () {
+			const account = generateAccount();
+			let hexStr = "62fdfc072182654f163f5f0f9a621d729566c74d0aa413bf009c9800418c19cd";
+			let data = Buffer.from(
+				hexStr,
+				"hex"
+			);
+			const signature = signBytes(data, account.sk);
+			//flip a bit and it should not pass
+			hexStr = "52fdfc072182654f163f5f0f9a621d729566c74d0aa413bf009c9800418c19cd"
+			data = Buffer.from(
+				hexStr,
+				"hex"
+			);
+			stack.push(data); 
+			stack.push(signature); 
+			stack.push(decodeAddress(account.addr).publicKey);
+
+			const op = new Ed25519verify_bare([], 1);
+			op.execute(stack);
+			const top = stack.pop();
+			assert.equal(top, 0n);
+		});
+
+		it(
+			"should throw an invalid type error",
+			execExpectError(
+				stack,
+				["1", "1", "1"].map(BigInt),
+				new Ed25519verify_bare([], 1),
+				RUNTIME_ERRORS.TEAL.INVALID_TYPE
+			)
+		);
+
+		it(
+			"should throw an error if stack is below min length",
+			execExpectError(
+				stack,
+				[],
+				new Ed25519verify_bare([], 1),
+				RUNTIME_ERRORS.TEAL.ASSERT_STACK_LENGTH
+			)
+		);
+
+		it("Should return correct cost", () => {
```

Comment:
Could we follow one convention in all the tests? Lets use a capital letter for all of them eg. `Should1`